src/dotnet-install.sh

```diff
@@ -1213,13 +1214,13 @@ generate_akams_links() {
 
         if [[ "$dry_run" == true ]]; then
             print_dry_run "$download_link" "" "$effective_version"
-            return 1
+            return 127
         fi
 
         #  Check if the SDK version is already installed.
         if is_dotnet_package_installed "$install_root" "$asset_relative_path" "$effective_version"; then
             say "$asset_name with version '$effective_version' is already installed."
-            return 1
+            return 127
         fi
```

Comment:
suggestion: can we move this block to main function?
it is repeated in `generate_akams_links()` and `generate_regular_links()` and somewhat hidden.
If this block is moved, we don't need custom error codes.